tendrl/node_agent/manager/manager.py

```diff
@@ -239,6 +246,31 @@ def on_pull(self, raw_data):
                     self.etcd_client.write(("nodes/%s/Disks/free/%s") % (
                         raw_data["node_id"], disk), "")
 
+    def load_and_execute_discovery_plugins(self, node_id):
+        # platform plugins
+        LOG.info("load_and_execute_discovery_plugins, platform plugins")
+        pMgr = PlatformManager()
+        pMgr.load_plugins()
```

Comment:
During object creation of PlatformManager() a method load_plugins is called from __init__, why again this function is called?